spec/std/json/serialization_spec.cr

```diff
@@ -215,6 +215,19 @@ describe "JSON serialization" do
       "\u{19}".to_json.should eq("\"\\u0019\"")
     end
 
+    it "does for String with control codes in a few places" do
+      "\fab".to_json.should eq("\"\\fab\"")
```

Comment:
It's probably much more readable to use `%q("\fab")` here instead of `"\"\\fab\""`.

Reply:
fixed